charts/models/values.yaml

```diff
@@ -161,6 +161,18 @@ catalog:
     # You can also use nvidia-gpu-a100-80gb:8
     resourceProfile: nvidia-gpu-h100:8
     targetRequests: 500
+  llama-3.1-70b-instruct-awq-int4-gh200:
+    enabled: false
+    features: [TextGeneration]
+    url: hf://hugging-quants/Meta-Llama-3.1-70B-Instruct-AWQ-INT4
+    engine: VLLM
+    args:
+      - --max-model-len=16384
+      - --max-num-batched-token=16384
+      - --enable-prefix-caching
+      - --disable-log-requests
+    resourceProfile: nvidia-gpu-gh200:1
+    targetRequests: 50
```

Comment:
Let's mergethis as-is. We can optimize later. 

Here are the are the settings I would suggest trying out as well:
```
  args:
      - --max-num-seqs=1024
      - --kv-cache-dtype=fp8
      - --enforce-eager
      - --enable-chunked-prefill
      - --num-scheduler-steps=8
targetRequests: 1024
```
not sure if those settings work with awq, but I've had great luck with the fp8 models published by Neural Magic. You should be able to fit fp8 model on a single gh200 and have decent context length if you enable --kv-cache-dtype=fp8

Reply:
I attempted to use an FP8 with the provided args, but it is not working properly. I think it might be best to add that model separately outside of this PR.